tests/unit/stats/test_view_data.py

```diff
@@ -89,7 +90,7 @@ def test_record(self):
         value = 1
         self.assertEqual({}, view_data.tag_value_aggregation_data_map)
 
-        view_data.record(context=context, value=value, timestamp=time)
+        view_data.record(context=context, value=value, timestamp=time, attachments=None)
```

Comment:
Please also add test cases where `attachments` is not passed.

Reply:
Done